pages/premium_shop.php

```diff
@@ -469,7 +469,11 @@ function premiumShop(): void {
                 if (!isset($_POST['change_forbidden_seal'])) {
                     // Convert remaining premium time to days and calculate AK value
                     $akCredit = $player->forbidden_seal->calcRemainingCredit();
-                    $new_seal = new ForbiddenSeal($system, $seal_level);
+                    $new_seal = ForbiddenSeal::fromDb(
+                        system: $system,
+                        seal_level: $seal_level,
+                        seal_end_time: time() + ForbiddenSeal::SECONDS_IN_DAY
```

Comment:
This seems a little problematic that we're putting in a fake value in a lot of different places - Could we make `seal_end_time` nullable? Then we have a clear understanding that if there's no `seal_end_time`, this isn't representing an active player's seal